quizzes/midterm.md

```diff
@@ -162,13 +256,25 @@
 	    "water"
 	  end
 	end
-	```
+```
 
-1. What is the difference between using `extend` and `include`?
+```lang-ruby
+	class Monkey < Animal
+		def eat 
+			"Bannans are yummy."
+		end
+	end
+```
+
+26: What is the difference between using `extend` and `include`?
```

Comment:
0/10 points.

Answer
`extend` adds methods from the module as class methods, `include` adds methods from the module as instance methods.